src/org/usfirst/frc/team2706/robot/subsystems/GearHandler.java

```diff
@@ -0,0 +1,39 @@
+package org.usfirst.frc.team2706.robot.subsystems;
+
+import org.usfirst.frc.team2706.robot.RobotMap;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/** 
+ * Coordinates commands for the Gear Handler Arm mechanisms. 
+ */
+public class GearHandler extends Subsystem {
+
+    private DoubleSolenoid solenoid = new DoubleSolenoid(RobotMap.SOLENOID_FORWARD_CHANNEL, RobotMap.SOLENOID_REVERSE_CHANNEL);
+    
+    // Let's use this to keep track of whether the arm is closed :)
+    private boolean closed = true;
+
+    public void initDefaultCommand() {
+    }
+    
+    public void OpenArm() {
```

Comment:
In Java, the naming convention for methods is camel case, similar to variables.
`openArm()`